### DISCUSSION TITLE
Idea: High Quality Audio

### DISCUSSION THREAD

**Discussion by mt-evvi** [ScratchAddons/ScratchAddons#5184] [Ideas]
I wanted to make a few Projects already with higher quality Music, but Scratch always compresses the Audio to way low Quality Settings

- **So maybe an Addon similar to the Image Quality for Audio could be made.**

I know Scratch wants to keep the File size down, but 2mb? Really? Wish i could have th full Sound Quality.
Thanks if this happens.

**Comment by penguinmoose** [comment]
This would be useful, but the high quality music has to be stored _somewhere_. Scratch has a size limit, and an addon can’t fix that.

You can try making your music shorter or choosing a compressed file format like mp3.

**Comment by GarboMuffin** [comment]
Should note that if you don't edit the sound in the Scratch sound editor, I'm pretty sure the sound stored in the project will be exactly the one you upload

**Reply by WorldLanguages** [reply]
Is this the case?
I think 2.0 compressed the file automatically, even if you didn't edit it

**Reply by GarboMuffin** [reply]
I just tested it - as long as you don't edit the file in the sound editor, the file that gets stored in the project is bit-for-bit identical to the file you upload.

**Reply by WorldLanguages** [reply]
Interesting.
I have read many times that Scratch 2.0 used to compress sound files intentionally, to avoid copyright issues by being a possible place to host high-quality audio. I guess it was a myth.
